MANIFEST.in

```diff
@@ -1,6 +1,2 @@
-include notifications_utils/email_preview_template.jinja2
-include notifications_utils/email_template.jinja2
-include notifications_utils/letter_pdf_template.jinja2
-include notifications_utils/letter_preview_template.jinja2
-include notifications_utils/sms_preview_template.jinja2
+recursive-include notification_utils *.jinja2
```

Comment:
(for reference you can test this by running `python setup.py sdist` and taking a look at the logs it outputs or look in the resultant tar file

`warning: no files found matching '*.jinja2' under directory 'notification_utils'`